### PR TITLE
Get repo ready for transferring to romgar.

### PR DESCRIPTION
**Pull request by smn**
I'm not at all active on this project anymore, @romgar's been doing a stellar job at working on this and I'd like to transfer the repo to him.

If anyone has any objections then please shout.
The only thing I'd probably want to look at is adding an AUTHORS file to keep track of contributions made over time.

### CODE REVIEW

**Comment by smn** [comment]
We may want to switch this code base to semver.org style of versioning, I'd also prefer we pin a version at which I transfer the repo to you for historical reference.

Thoughts?

File: setup.py [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+    license='BSD',
+    description=("Tracking dirty fields on a Django model instance "
+                 "(actively maintained)"),
+    long_description=open('README.rst', 'r').read(),
+    author='Romain Garrigues',
+    packages=find_packages('src'),
+    package_dir={'': 'src'},
+    install_requires=listify('requirements.txt'),
+    classifiers=listify('CLASSIFIERS.txt')
 )
```
Comment (smn) [inline setup.py]: Most of these changes are pep8 fixes. The real change was me putting @romgar as the author.

**Comment by romgar** [comment]
@smn, you are talking about semver.org because you think that I was a little too fast for minor versions ?
We can set the `0.7` as the version transfer, no more issues/PR, a fresh new start :-)

**Comment by smn** [comment]
I've transferred the repo owernship to @romgar, I'll wrap up this PR to fix any lingering references.

**Comment by romgar** [comment]
Thanks @smn !! I will take care of it !!

**Comment by romgar** [comment]
Hey @smn, I want to release a new version of `django-dirtyfields`. Can I merge this PR, as I need to update the `setup` file for the new version ?

**Comment by smn** [comment]
Please go ahead.

On 30 October 2015 at 19:27, Romain Garrigues <notifications@github.com>
wrote:

> Hey @smn <https://github.com/smn>, I want to release a new version of
> django-dirtyfields. Can I merge this PR, as I need to update the setup
> file for the new version ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/romgar/django-dirtyfields/pull/45#issuecomment-152594162>
> .
>



-- 
Simon de Haan | Chief Engineer Praekelt Foundation

praekeltfoundation.org | praekelt.com
M: +27 (0) 76 449 3806  |  http://twitter.com/smn

**Pull request merged**